common/api/core-backend.api.md

```diff
@@ -2680,18 +2680,30 @@ export interface ExportGraphicsMesh {
     points: Float64Array;
 }
 
+// @public
+export class ExportGraphicsMeshSubsetVisitor extends ExportGraphicsMeshVisitor {
+    static createSubsetVisitor(polyface: ExportGraphicsMesh, facetIndices: number[], numWrap?: number): ExportGraphicsMeshSubsetVisitor;
+    getVisitableFacetCount(): number;
+    moveToNextFacet(): boolean;
+    moveToReadIndex(subsetIndex: number): boolean;
+    parentFacetIndex(subsetIndex?: number): number | undefined;
+    reset(): void;
+}
+
 // @public
 export class ExportGraphicsMeshVisitor extends PolyfaceData implements PolyfaceVisitor {
+    protected constructor(facets: ExportGraphicsMesh, numWrap: number);
     clearArrays(): void;
     clientAuxIndex(_i: number): number;
     clientColorIndex(_i: number): number;
     clientNormalIndex(i: number): number;
     clientParamIndex(i: number): number;
     clientPointIndex(i: number): number;
-    clientPolyface(): Polyface;
-    static create(polyface: ExportGraphicsMesh, numWrap: number): ExportGraphicsMeshVisitor;
+    clientPolyface(): Polyface | undefined;
```

Comment:
This changes the return type of a `@public` method in a breaking way.

Reply:
Even when we always return `undefined` typecast away?

Reply:
Definition of a breaking API change = previously valid code produces a compilation error when I upgrade to new version.

My existing code compiles and works:
```ts
  console.log(meshVisitor.clientPolyface.twoSided);
```

When I upgrade to the new version I'll suddenly get an error because `meshVisitor.clientPolyface` may now be undefined.

Reply:
I guess I don't understand since this visitor always returns `undefined` for `clientPolyface`. I admit the double cast to `any` and `Polyface` is mysterious. But it's `undefined` under the hood. How does your code run?

Reply:
> `public clientPolyface(): Polyface { return (undefined as unknown) as Polyface; }`

Wow, what the heck was the point of that.

I am reviewing the api.md files without looking at implementation. Yes, in this case the old API could never have worked, so breaking it = fixing it.